cg/cli/compress.py

```diff
@@ -53,10 +53,13 @@ def bam(context, case_id, number_of_conversions, ntasks, mem, dry_run):
         case_has_bam_file = True
         for sample, bam_files in bam_dict.items():
             bam_path = Path(bam_files["bam"].full_path)
-            if not context.obj["crunchy"].bam_compression_possible(bam_path=bam_path):
+            if not context.obj["crunchy"].bam_compression_possible(
+                bam_path=bam_path
+            ) or context.obj["crunchy"].cram_compression_pending(bam_path=bam_path):
```

Comment:
Maybe you should split these 2 condition into separate ifs so that you can test them individually and have explicit log messages i.e. separate possible from pending

Reply:
yes thats would be better. Fixed that!